### DISCUSSION TITLE
Removing tag 347, JPEGTables

### DISCUSSION THREAD

**Discussion by johneightyone** [dlemstra/Magick.NET#1354] [Help] [answered]
Can anyone tell me if it's possible, and how, to NOT include JPEG tables for a JPEG compressed TIF file?
I've tried various ChatGPT suggestions, but none seem to work.

**Comment by dlemstra** [comment, marked as answer]
This is not yet possible but you will be able to do that in the next release (13.0.0 planned for 2023-03-13):

```C#
new TiffWriteDefines
{
    JpegTablesMode = TiffJpegTablesMode.None
};
```
